src/app.ts

```diff
@@ -130,4 +133,10 @@ void botManager.start(options).asCallback(err => {
     if (err) {
         throw err;
     }
+
+    void httpManager.start().asCallback(err => {
+        if (err) {
+            throw err;
+        }
+    });
 });
```

Comment:
I think might be better to never initiate HttpManager class from the beginning if it's disabled.

maybe like:
```ts
void botManager.start(options).asCallback(err => {
    if (err) {
        throw err;
    }
    
    if (options.enableHttpApi) {
        const httpManager = new HttpManager(options);

        void httpManager.start().asCallback(err => {
            if (err) {
                throw err;
            }
        }
    });
});
```

Reply:
https://github.com/TF2Autobot/tf2autobot/pull/413/commits/6a67423fe3df5358d4ef0000fce7723e60699248